src/components/Nav/Nav.container.js

```diff
@@ -35,8 +39,16 @@ export const NavContainer = (props) => {
     Fetchdata();
   });
 
+  const handleClickActiveTab = (tabId) => {
+    setActiveNavigationTab(tabId);
+    BrowserDatabase.setItem('activeTabId', tabId);
```

Comment:
Remove this data from local storage on LOGOUT. So, mainpage will be always default when you log in